bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -56,13 +56,16 @@ spec:
           trap quit SIGTERM
           /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --system-id=random
           ovs-appctl vlog/set "file:${OVS_LOG_LEVEL}"
+          /usr/share/openvswitch/scripts/ovs-ctl --protocol=udp --dport=${GENEVE_PORT} enable-protocol
```

Comment:
@pecameron good catch, though we also need to pass GENEVE_PORT into ovnkube --node in the ovnkube node container as well.

Reply:
@dcbw, created cno PR 345 for above. ovnkube encap-port should wait for sdn-456 (cno pr 333) to land.